app/Http/Controllers/HomeController.php

```diff
@@ -42,7 +44,48 @@ public function bbcodePreview()
 
     public function getChangelog()
     {
-        return view('home.changelog');
+        $streamId = get_int(Request::input('stream_id'));
+        $build = Request::input('build');
+
+        if ($streamId && $build) {
+            // in this case ignore the build and show page for the stream id
+            $build = null;
+        } elseif (!$streamId && !$build) {
+            $streamId = config('osu.changelog.featured_stream');
+        }
+
+        $streamIds = implode(',', config('osu.changelog.update_streams'));
+
+        $builds = Build::orderBy('date', 'desc')
+            ->take($build ? 1 : config('osu.changelog.build_count'));
+
+        if ($streamId) {
+            $builds->where('stream_id', $streamId);
+        } elseif ($build) {
+            $builds->where('version', $build);
```

Comment:
am I reading it right this goes like:

- query `Build`
- filter where `version: $build`
- pluck `version`

?

Might as well just use `$build` variable itself directly?